composer.json

```diff
@@ -30,7 +30,8 @@
     },
     "scripts": {
         "test": "phpunit -v -c phpunit.xml",
-        "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
+        "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating",
+        "run-cs": "@php ./vendor/bin/php-cs-fixer fix -v --dry-run --show-progress=estimating"
```

Comment:
```suggestion
        "run-cs": "php-cs-fixer fix -v --dry-run --show-progress=estimating"
```

Reply:
Some packages use `check-cs`, which version should we stick to to prevent mismatch issues?

Reply:
Looks like `check-cs` is more popular than `php-cs`:
<img width="564" alt="Zrzut ekranu 2021-06-18 o 10 49 24" src="https://user-images.githubusercontent.com/10993858/122534492-dbcc5900-d022-11eb-8f59-fa7b7df79c1b.png">

<img width="561" alt="Zrzut ekranu 2021-06-18 o 10 48 26" src="https://user-images.githubusercontent.com/10993858/122534342-b6d7e600-d022-11eb-8a23-4349ad5b198a.png">

If you're ok with it I'd go with `check-cs` here as well (and maybe, as a follow-up, change the `php-cs` occurrences into `check-cs` to keep things consistent across packages).